src/Directory.Build.props

```diff
@@ -31,6 +31,7 @@
     <TestsBuildTFMs>net6.0-windows10.0.19041.0;net7.0-windows10.0.19041.0</TestsBuildTFMs>
     <TestsBuildTFMs Condition="'$(CIBuildReason)' == 'CI'">net6.0-windows10.0.19041.0;net7.0-windows10.0.19041.0</TestsBuildTFMs>
     <WindowsAppSDKVerifyWinrtRuntimeVersion>false</WindowsAppSDKVerifyWinrtRuntimeVersion>
+    <DisableRuntimeMarshalling>True</DisableRuntimeMarshalling>
```

Comment:
I don't think this actually does anything. You need to define the attribute, no? You can use this:

```xml
<!-- Emit the [DisableRuntimeMarshalling] attribute for all .NET 8 projects -->
<ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
  <AssemblyAttribute Include="System.Runtime.CompilerServices.DisableRuntimeMarshallingAttribute" />
</ItemGroup>
```

Reply:
https://github.com/dotnet/sdk/pull/34388 :)

Reply:
Ooh really cool, I had completely missed this! Thank you for adding it! 😄
It would seem I have some PRs to make in my own repo then eheheh